snapcraft/parts/lifecycle.py

```diff
@@ -449,7 +490,9 @@ def _run_in_provider(
         except subprocess.CalledProcessError as err:
             capture_logs_from_instance(instance)
             raise errors.SnapcraftError(
-                f"Failed to execute {command_name} in instance."
+                f"Failed to execute {command_name} in instance. "
+                "Run the same command again with --debug to shell into "
```

Comment:
this should use the extra params from CraftError which SnapcraftError inherits from